cmd/ratify/cmd/verify.go

```diff
@@ -99,7 +99,8 @@ func verify(opts verifyCmdOptions) error {
 		return err
 	}
 
-	verifiers, err := vf.CreateVerifiersFromConfig(cf.VerifiersConfig, config.GetDefaultPluginPath())
+	emptyNamespace := ""
```

Comment:
nit: make this a global constant?

Reply:
will move to https://github.com/deislabs/ratify/blob/main/internal/constants/constants.go